<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>461</width>
      <height>411</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>459</width>
       <height>409</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionAc"/>
   </widget>
   <widget class="QMenu" name="menuG_ster">
    <property name="title">
     <string>Göster</string>
    </property>
    <addaction name="actionYakinlastir"/>
    <addaction name="actionUzaklastir"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuG_ster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAc"/>
   <addaction name="separator"/>
   <addaction name="actionUzaklastir"/>
   <addaction name="actionYakinlastir"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionYakinlastir">
   <property name="text">
    <string>Yakınlastır</string>
   </property>
  </action>
  <action name="actionUzaklastir">
   <property name="text">
    <string>Uzaklastir</string>
   </property>
  </action>
  <action name="actionAc">
   <property name="text">
    <string>Aç</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
